source/java/product-guide/index.markdown

```diff
@@ -1757,14 +1757,14 @@ For example, assume that an account store represented by mapping `$accountStoreM
 
 Applications cannot store Accounts directly - Accounts are always stored in an Account Store (a Directory or Group). Therefore, if you would like an application to be able to create new accounts, you must specify which of the application's associated account stores should store the application's newly created accounts.  This designated account store is called the application's _default account store_.
 
-You specify an application's default account store by setting the AccountStoreMapping's `isDefaultAccountStore` attribute to equal `true`.  You can do this when you create the `accountStoreMapping` resource.  Or if the resource has already been created:
+You specify an application's default account store by setting the ApplicationAccountStoreMapping's `isDefaultAccountStore` attribute to equal `true`.  You can do this when you create the `applicationAccountStoreMapping` resource.  Or if the resource has already been created:
```

Comment:
Remove "equal" in "...attribute to equal `true`...". Therefore it should be like this "...attribute to `true`...."